### DISCUSSION TITLE
Pre-trained SFT/RM checkpoint

### DISCUSSION THREAD

**Discussion by hijkzzz** [OpenRLHF/OpenRLHF#57] [Announcements]
https://huggingface.co/chuyi777/openllama2_checkpoint

pretrain model: llama2-7b

* rm-anthropic_hh-lmsys-oasst-webgpt.pt (accuracy 0.75)
* sft-alpaca.pt
* sft-ocra-500k.pt
